exercises/practice/forth/.meta/config.json

```diff
@@ -1,6 +1,59 @@
 {
   "blurb": "Implement an evaluator for a very simple subset of Forth",
   "authors": [],
```

Comment:
another empty one, added in https://github.com/exercism/elixir/commit/a3382372a1ed71550466844cd310af18645fc340